common/source/docs/common-paramosd.rst

```diff
@@ -0,0 +1,146 @@
+
+The RC transmitter's roll and pitch stick input is used to navigate through the screen, pick 
+a parameter and adjust the parameter value. Parameter values can be adjusted in disarmed state only. 
+
+Additionally there's a "save and reboot" button on each screen that allows to save the adjusted settings and reboot the 
+flightcontroller if required. It defaults to "reboot", will auto-switch to "save" if a parameter value has been altered, 
+and switch back to "reboot" after saving is completed. 
+
+.. note::
+   
+   Using the "save" button after adjusting a parameter will make the adjustments persistent, otherwise the value will be reset to its prior state after a reboot.
+
+Accessing parameter screens
+===========================
+
+To enable the parameter screens, set :ref:`OSD5_ENABLE <OSD5_ENABLE>` and :ref:`OSD6_ENABLE <OSD6_ENABLE>` = 1.
+Parameter screens can be accessed in the same way the conventional osd screens are accessed. 
+See  `_screens-and-switching: <common-osd-overview.html#screens-and-screen-switching>` for reference.
+When one of the parameter screens is selected, adjustments can be made in disarmed state only. If a parameter screen is 
+selected inflight, the screen allows display only. This allows live monitoring of automatic parameter adjustments, 
+i.e. when performing autotune procedures or automatic servo trims.
+
+
+Menu navigation
+===============
+
+Use the RC transmitter's roll and pitch input to navigate through the parameter screen. UP / DOWN and LEFT / RIGHT will navigate through 
+the screen's parameter slots. The selected parameter will blink. Move right on the selected parameter to adjust its value using 
+up or down stick input. Leave the selection with left roll stick input. On a typical mode 2 type transmitter this would be the right 
+gimbal stick:
+
+.. image:: ../../../images/stickinput.jpg
+    :target: ../_images/stickinput.jpg
+    :width: 450px
+
+  
+Parameter slot scroll-selection
+===============================
+
+Holding roll right for > 3 seconds on a parameter's value will highlight both the selected parameter and its value (blink). This allows 
+you to scroll through the full list of parameters and place another paramater into that respective slot using pitch up and down stick input. 
+Move left to exit parameter selection and get back into standard screen navigation.
+
+.. note::
+
+   OSD based parameter adjustment is designed to allow basic adjustments without any aditional data link or GCS setup required. However, 
+   some parameters with exceptionally wide value ranges, as well as complex bitmap-type parameters might not be ideal for adjustment using parameter OSD.
+
+
+Parameter range, increment and type-mnemonics
+=============================================
+
+OSD based parameter menu uses parameter metadata to estimate a useful value range and incrementation for each parameter. Additionally, 
+this supports to display human-readable abbreviations for: 
```

Comment:
suggestion:
Additionally, support for human-readable abbreviations is provided for:

Reply:
Revised suggestion:
Additionaly, support is provided to be able to display some parameter values in more readable form by the use of some mnemonic sets for certain parameters:

missed spelling on failsafe in that list

Reply:
fixed